_data/projects/cdnjs.yml

```diff
@@ -0,0 +1,14 @@
+name: cdnjs 
+desc: The most famous free and open source web front-end libraries cdn
+site: https://cdnjs.com
+tags:
+- cdn
+- js
+- css
+- libraries
+- web
+- frontend
+- javascript
+upforgrabs:
+  name: issues
+  link:　https://github.com/cdnjs/cdnjs/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22wait+for+response%22+-label%3A%22in+progress%22+
```

Comment:
I wonder if is possible to add maybe single quote to that string, maybe we can work together on this part, because I need a much more complex rule to filter out some in progress label, and it's good to support it, if you don't mind, maybe using goo.gl to shorten the url is a choice, many thanks for your review.